cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/server/GameServer.java

```diff
@@ -130,6 +131,100 @@ public Path getDir() {
+                if (!Files.exists(Paths.get("local/cache/web_plugins/" + plugin.getName() + ".jar"))) {
+                    try {
+                        URLConnection urlConnection = new URL(plugin.getUrl()).openConnection();
+                        urlConnection.setRequestProperty("User-Agent",
+                                                         NetworkUtils.USER_AGENT);
+                        Files.copy(urlConnection.getInputStream(), Paths.get("local/cache/web_plugins/" + plugin.getName() + ".jar"));
+                    } catch (Exception ex) {
+                        ex.printStackTrace();
+                    }
+                }
+            }
+            break;
+            case MASTER: {
+                if (!Files.exists(Paths.get("local/cache/web_plugins/" + plugin.getName() + ".jar")) && CloudNetWrapper.getInstance()
+                                                                                                                       .getSimpledUser() != null) {
+                    try {
+                        URLConnection urlConnection = new URL(
+                            String.format("http://%s:%d/cloudnet/api/v1/download",
+                                          CloudNetWrapper.getInstance().getWrapperConfig().getCloudnetHost(),
+                                          CloudNetWrapper.getInstance().getWrapperConfig().getWebPort())).openConnection();
+                        urlConnection.setRequestProperty("User-Agent", NetworkUtils.USER_AGENT);
+
+                        SimpledUser simpledUser = CloudNetWrapper.getInstance().getSimpledUser();
+                        urlConnection.setRequestProperty("-Xcloudnet-user", simpledUser.getUserName());
+                        urlConnection.setRequestProperty("-Xcloudnet-token", simpledUser.getApiToken());
+                        urlConnection.setRequestProperty("-Xmessage", "plugin");
+                        urlConnection.setRequestProperty("-Xvalue", plugin.getName());
+
+                        urlConnection.connect();
+                        CloudNetWrapper.getInstance().getCloudNetLogging().log(Level.INFO,
+                                                                               String.format("Downloading %s.jar", plugin.getName()));
+                        Files.copy(urlConnection.getInputStream(), Paths.get("local/cache/web_plugins/" + plugin.getName() + ".jar"));
+                        CloudNetWrapper.getInstance().getCloudNetLogging().log(Level.INFO, "Download was completed successfully!");
+                    } catch (Exception ex) {
+                        ex.printStackTrace();
+                    }
+                }
+            }
+            break;
+            default:
+                break;
+        }
+    }
+
+    /**
+     * Copy the cloudnet api from wrapper to server folder.
+     */
+    private void copyCloudNetApi() {
+        try {
+            Files.deleteIfExists(Paths.get(path + "/plugins/CloudNetAPI.jar"));
```

Comment:
Use the varargs to your advantage
```suggestion
            Files.deleteIfExists(Paths.get(path, "plugins", "CloudNetAPI.jar"));
```